frontend/src/Student/components/PhotoSubmissionForm.js

```diff
@@ -240,14 +240,10 @@ class PhotoSubmissionForm extends Component {
                       participants: values.groupParticipants
                     }
                     : null,
-<<<<<<< HEAD
```

Comment:
Why dont we have this conflict in other media submission?

Reply:
I think other media is missing the displayName field here don't we need that?

Reply:
I think `displayName` popped in above the `hometown` field in this file, so the delete conflict didn't pop in.